src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -2222,6 +2258,33 @@ private JSONObject googleGlobalFailureReport(JSONObject statusObj, JSONObject de
 		{
 			log.error("Problem parsing site members for site id = " + siteId + " " + e.getStackTrace());
 		}
+		addMembers.put(Utils.REPORT_ATTR_ITEMS, addMembers_items);
+		
+		// insert the counts element
+		addMembers_counts.put(Utils.REPORT_ATTR_COUNTS_SUCCESSES, count_success);
+		addMembers_counts.put(Utils.REPORT_ATTR_COUNTS_ERRORS, count_error);
+		addMembers.put(Utils.REPORT_ATTR_COUNTS, addMembers_counts);
+		
+		// insert the status element
+		String membership_status = Utils.REPORT_STATUS_OK;
+		if (count_success == 0 && count_error > 0)
+		{
+			// all failures
+			membership_status = Utils.REPORT_STATUS_ERROR;
+		} else if (count_success > 0 && count_error > 0)
+		{
+			// partial failures
+			membership_status = Utils.REPORT_ATTR_COUNT_PARTIALS;
+		}
+		addMembers.put(Utils.REPORT_ATTR_STATUS, membership_status);
+		
+		// the migration status JSON object shall contain a "details" element
+		// which list the details of membership transfers
+		JSONObject statusJSON = new JSONObject();
+		JSONObject detailsJSON = new JSONObject();
+		detailsJSON.put(Utils.REPORT_ATTR_ADD_MEMBERS, addMembers);
```

Comment:
The error message says "Problem adding user = <uniqname> with role = <role name> to Box folder id = <Box folder id>. Then API returned an error code: 400"

So I think we are good here.